encoding/mvt/prepare.go

```diff
@@ -117,7 +101,7 @@ func scalePolygon(g geom.Polygon, tile *tegola.Tile) (p geom.Polygon) {
 	}
 
 	for _, line := range lines.LineStrings() {
-		ln := scalelinestr(line, tile)
+		ln := preparelinestr(line, tile)
 		if len(ln) < 2 {
 			if debug {
```

Comment:
remove the debug pieces from this package

Reply:
This would be important if to know why a line is getting skipped. I think you should keep the debug piece. The convention in the geom package is to go have a  debug.go file that allows one to turn own extra logging and turn it off as needed. I think this is fine.